clients/src/main/java/org/apache/kafka/common/requests/CreateTopicsResponse.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.requests;
+
+
+import org.apache.kafka.common.protocol.ApiKeys;
+import org.apache.kafka.common.protocol.Errors;
+import org.apache.kafka.common.protocol.ProtoUtils;
+import org.apache.kafka.common.protocol.types.Schema;
+import org.apache.kafka.common.protocol.types.Struct;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class CreateTopicsResponse extends AbstractRequestResponse {
+    private static final Schema CURRENT_SCHEMA = ProtoUtils.currentResponseSchema(ApiKeys.CREATE_TOPICS.id);
+
+    private static final String TOPIC_ERROR_CODES_KEY_NAME = "topic_error_codes";
+    private static final String TOPIC_KEY_NAME = "topic";
+    private static final String ERROR_CODE_KEY_NAME = "error_code";
+
+    /**
+     * Possible error codes:
+     *
+     * INVALID_TOPIC_EXCEPTION(17)
+     * CLUSTER_AUTHORIZATION_FAILED(31)
+     * TOPIC_ALREADY_EXISTS(32)
+     * INVALID_PARTITIONS(38)
+     * INVALID_REPLICATION_FACTOR(39)
+     * INVALID_REPLICA_ASSIGNMENT(40)
+     * INVALID_CONFIG(41)
+     * NOT_CONTROLLER(42)
```

Comment:
Well technically we can't. There is no way using the Java CreateTopicsRequest class that you can generate and invalid request. I can list it for completeness though.